platform-jobs/samza/course-certificate-generator/src/main/java/org/sunbird/jobs/samza/service/util/CertificateGenerator.java

```diff
@@ -452,7 +455,7 @@ private void updatedES(String index, String type, Map<String, Object> dataToUpda
     private void populateCreatedCertificate(List<Map<String, String>> updatedCerts, Map<String, Object> certificate, String certificateName, Date issuedOn, boolean reIssue) {
         updatedCerts.add(new HashMap<String, String>(){{
             put(CourseCertificateParams.name.name(), certificateName);
-            put(CourseCertificateParams.id.name(), (String) certificate.get(CourseCertificateParams.id.name()));
+            put(CourseCertificateParams.identifier.name(), (String) certificate.get(CourseCertificateParams.id.name()));
             put(CourseCertificateParams.url.name(), (String) certificate.get(CourseCertificateParams.pdfUrl.name()));
```

Comment:
Below line also has to be deleted:
put(CourseCertificateParams.url.name(), (String) certificate.get(CourseCertificateParams.pdfUrl.name()));